amp/audits.json

```diff
@@ -0,0 +1,112 @@
+[
+  {
+    "id": "unused-css-rules",
+    "title": "Defer unused CSS",
+    "description": "Remove unused rules from stylesheets to reduce unnecessary bytes consumed by network activity. [Learn more](https://developers.google.com/web/tools/lighthouse/audits/unused-css).",
+  },
+  {
+    "id": "uses-webp-images",
+    "title": "Serve images in next-gen formats",
+    "description": "Image formats like JPEG 2000, JPEG XR, and WebP often provide better compression than PNG or JPEG, which means faster downloads and less data consumption. [Learn more](https://developers.google.com/web/tools/lighthouse/audits/webp).",
+    "secondaryDescription": "Consider displaying all your amp-img components in WebP formats while specifying an appropriate fallback for other browsers. [Learn more](https://www.ampproject.org/docs/reference/components/amp-img#example:-specifying-a-fallback-image)."
+  },
+  {
+    "id": "offscreen-images",
```

Comment:
Is it possible to disable specific audits? For example, this audit is not actionable for AMP as devs can't influence this behaviour.

Reply:
Unfortunately don't think it would be possible to disable some audits that aren't applicable. But since this is the default behavior w/ AMP, it should always show as a passing audit.

I think it might be worthwhile to have something like 👍 

    With AMP, `amp-img` elements are lazy loaded by default.

It's not directly actionable, but it lets the user know it's already taken care of.

Wdyt?

Reply:
I think that's OK. It might be a bit confusing for users if lighthouse still flags this.

Reply:
Hmm that's a good point. But ideally, Lighthouse shouldn't ever show this as a failing audit if images are lazy loaded by default so we might be okay with something not directly actionable like this/